src/content/reference/fleet-management/migration_to_cluster_api.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Migration to Cluster API
+description: A explanation on how the migration from our old Vintage management clusters to Cluster API works.
+weight: 20
+last_review_date: 2024-03-14
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - What are the requirements for migrating a cluster to Cluster API?
+  - What are the recommendations for a smooth migration?
+---
+
+In Giant Swarm we have used Kubernetes to build platforms since our beginning. We started with a custom solution to manage the lifecycle of the clusters due to the lack of a standard way out of there. It has worked well for us and our customers, but we learnt many companies were resolving the same problem over and over again, so we foster and push for joining forces and build a standard approach. [Cluster API]({{< relref "/reference/fleet-management/introduction" >}}) is backed by the Kubernetes community and covers different providers like AWS, Azure, GCP, and others.
+
+In this document we explain the migration path from our vintage system to the [Cluster API](https://cluster-api.sigs.k8s.io/) standard. This document provides an overview of the process, its requirements, and recommendations to ensure a successful migration.
+
+## Goals
+
+Our new migration process looks for ensuring a smooth transition of your Kubernetes workload clusters from vintage generation to the new Cluster API (CAPI) flavour. This document provides an overview of the process, its requirements, and recommendations to ensure a successful migration.
```

Comment:
Maybe the previous paragraph could be integrated into this one because most of the information is duplicated.